### DISCUSSION TITLE
Inkstitch parameter

### DISCUSSION THREAD

**Discussion by Paillier** [inkstitch/inkstitch#2373] [Q&A]
Good day,
I just did my first try, I think I have correctly installed inkscape and inkstitch, but when I click on "parameter" in inkstitch nothing happens, I'm attaching a screenshot, it looks like parameter is empty... I don't I don't see a window opening and no point simulator appears.... can you help me? I have a French version. THANKS. Chris
![copie écran](https://github.com/inkstitch/inkstitch/assets/136691007/68bcabfa-501a-43f3-8c4b-e28cb7eb44d0)

**Comment by kaalleen** [comment]
Most probably this is the fault of your virus-program which decided to delete the Ink/Stitch executable. You may need to whitelist Ink/Stitch.
